units/commit_queue.go

```diff
@@ -265,80 +266,99 @@ func (j *commitQueueJob) processGitHubPRItem(ctx context.Context, cq *commitqueu
 		return
 	}
 
-	patchDoc, err := patch.MakeNewMergePatch(pr, projectRef.Id, evergreen.CommitQueueAlias, nextItem.MessageOverride)
-	if err != nil {
-		j.logError(err, "can't make patch", nextItem)
-		j.AddError(sendCommitQueueGithubStatus(j.env, pr, message.GithubStateFailure, "can't make patch", ""))
-		j.dequeue(cq, nextItem)
-		return
-	}
+	if nextItem.PatchId == "" {
```

Comment:
Is the idea here that, right now all PRs have PatchId == "", so this if statement is necessary, but after these changes are baked in all PRs will have PatchId set already, so they'll go into the else statement? I want to check my understanding of what the TODO is planning to delete.

Reply:
yep, after this is baked this block will just be an error because all pr entries should have patchId set